cli/enry/main.go

```diff
@@ -61,16 +61,22 @@ func main() {
 			return nil
 		}
 
-		content, err := ioutil.ReadFile(path)
-		if err != nil {
-			errors = true
-			log.Println(err)
-			return nil
-		}
-
-		language := enry.GetLanguage(filepath.Base(path), content)
-		if language == enry.OtherLanguage {
-			return nil
+		var language string
+		var ok bool
+		if language, ok = enry.GetLanguageByExtension(path); !ok {
```

Comment:
Oh, I see :S. Then it makes more sense to just define them outside the if statement:
```go
language, ok := enry..
if !ok {
}
```

Anyway, I'm quite happy with it as it is, just trying to understand go better. Thanks for the explanation!